cmake/InfoLoggerConfig.cmake.in

```diff
@@ -0,0 +1,16 @@
+@PACKAGE_INIT@
+
+get_filename_component(InfoLogger_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include(CMakeFindDependencyMacro)
+
+list(APPEND CMAKE_MODULE_PATH ${InfoLogger_CMAKE_DIR})
+
+find_dependency(Common HINTS /Users/awegrzyn/Common_install)
```

Comment:
I guess this is wrong, isn't it ?

BTW, I don't quite get why you need the get_filename_component  + list(append + list(remove_at) bits ? Looks like a "trick"

Reply:
Hi, Indeed it's not needed anymore